src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days === 7) {
+    return 230;
+  }
+
+  if (days === 6) {
```

Comment:
This condition is unnecessary since the logic should calculate the total cost dynamically rather than returning a hard-coded value for 6 days.